extensions/replication/tasks/ReplicateObject.js

```diff
@@ -139,17 +143,30 @@ class ReplicateObject extends BackbeatTask {
             sourceEntry.toFailedEntry(this.site);
         updatedSourceEntry.setReplicationSiteDataStoreVersionId(this.site,
             sourceEntry.getReplicationSiteDataStoreVersionId(this.site));
+        // TODO: understand when only metadata is updated!
+        const updateData = sourceEntry.getReplicationContent().includes('DATA');
         const kafkaEntries = [updatedSourceEntry.toKafkaEntry(this.site)];
         this.replicationStatusProducer.send(kafkaEntries, err => {
             if (err) {
-                log.error(
-                    'error in entry delivery to replication status topic',
-                    { method: 'ReplicateObject._publishReplicationStatus',
-                      topic: this.repConfig.replicationStatusTopic,
-                      entry: updatedSourceEntry.getLogInfo(),
-                      replicationStatus:
-                      updatedSourceEntry.getReplicationStatus(),
-                      error: err });
+                log.error('error in entry delivery to replication status topic', {
+                    method: 'ReplicateObject._publishReplicationStatus',
+                    topic: this.repConfig.replicationStatusTopic,
+                    entry: updatedSourceEntry.getLogInfo(),
+                    replicationStatus,
+                    error: err,
+                });
+                // TODO: Update failure metric when publish fails?
+            } else {
+                log.info('replication status published', {
```

Comment:
Previously calling `log.end()` allowed to have the `elapsed` time logged, is it still logged somewhere else? I understand we do not want to count in the elapsed time the time for kafka to return a delivery report since it does not relate to the execution time of the operation, but it's very important to be able to log the elapsed time of the replication operation.

I'd suggest to keep this log line as it was, and optionally add an error log if delivery fails (although it is likely logged somewhere else in the `BackbeatProducer` class), and a debug log when the delivery report is received successfully.